dallinger/command_line.py

```diff
@@ -277,16 +286,13 @@ def debug(verbose, bot):
     logfile = config.get('logfile')
     if logfile != '-':
         logfile = os.path.join(cwd, logfile)
-    config.extend({
-        "mode": u"debug",
-        "loglevel": 0,
-        "logfile": logfile
```

Comment:
Looks like this line to update the config with an absolute path to the logfile got lost. If it's not needed, we should remove the lines that set logfile above.